bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/runtime/ShellExecutorHelper.java

```diff
@@ -3,15 +3,20 @@
 import com.google.common.base.Optional;
```

Comment:
The other large part is in this class that actually makes the calls. Method signatures were changed to be able to use the command substitution maps.